client/src/store/reducer.ts

```diff
@@ -387,6 +389,42 @@ function onQuestionInputChanged(
   });
 }
 
+function findWebLinks(text: string): WebLink[] {
+  const REGEX_WEB_LINKS_ALL =
+    /(https?:\/\/(?:www.|(?!www))[^\s.]+\.[^\s]{2,}|www.[^\s]+.[^\s]{2,})/gi;
+
+  const webLinks: WebLink[] = (
```

Comment:
just return this directly (why define a variable?)

Reply:
https://github.com/mentorpal/mentor-client/pull/87/commits/fd8ed8ce5d7025b1b490402768b1a302f3de5083